CDCgov/prime-reportstream
# Flexion Onboarding UCSD

This PR adds the settings yml file for UCSD. UCSD only has a receiver for ORU messages. 

#### Note:
At this time, these settings will be used for testing the output of ingested HL7 ORU messages. The SFTP transport is used to facilitate testing. Eventually, the transport will change to REST, once connectivity with staging is performed.

Test Steps:
1. Run containers
2. Add settings to db by running this code: `./prime multiple-settings set --input ./settings/STLTs/CA/ucsd.yml`
3. Submit an HL7 message to API with the required routing values
4. Check if message was sent to the sftp folder

## Changes
- Added `/settings/staging/0170-ucsd-onboarding-receiver-settings.yml`

## Checklist

### Testing
